doc/about_us.md

```diff
@@ -0,0 +1,18 @@
+
+# About us
+
+## Origins
+
+OFT's roots go back to the year 2003 when it's first predecessor with the unimaginative name Requirement Manager first saw the light of day at 3SOfT GmbH in Erlangen Tennenlohe (Germany). Being a software supplier for the automotive industry, 3SOFT had a need for requirement tracing to fulfill the strict rules for safety-critical software.
+
+3SOFT was later acquired by the Finish Elektrobit group. And the Requirement Manager got a more sophisticated makeover in form of ReqM2. This had become necessary because the projects had become increasingly complex and required collecting coverage from multiple sub-modules. [Bernd "Poldi" Haberstumpf](https://github.com/poldi2015) improved the tracer over time. This led to the next incarnation dubbed "T-Reqs".
```

Comment:
I did the complete rewrite  in context of introduction of Autosar to have a multi level tracing 2024. The new version of the requirement was named ReqMgrNG.With the introduction of the ASIL-D Autosar version (microkernel) of the OS ReqM was born as stripped down ReqMgr missing a lot of functionality of the ReqMgrNG called ReqM. In parallel to activities 2004 T-Reqs a Java tool was created by a group of People but never used as far as I know.

Reply:
Maybe some additional background not relevant for the README. The ReqMgrNG is or was the tracing tool for Tresos Studio. The requirements.xml is the output format for ReqM2. It's somehow based on the Auotsar meta-format. ReqM2 became popular know at Elektrobit as we managed to place it is requirement tool as standard for EB.

Reply:
I am guessing you mean 2004 (obviously not 2024) when the rewrite happened.